src/conrollers/expenseController.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const userServices = require('../servises/userServices');
+const expenseServices = require('../servises/expenseServices');
+
+function getExpenses(req, res) {
```

Comment:
Use arrow functions instead of function declarations for consistency and shorter syntax.